public/app/features/live/centrifuge/service.ts

```diff
@@ -196,7 +197,16 @@ export class CentrifugeService implements CentrifugeSrv {
         }
 
         filtered.length = data.length;
-        subscriber.next({ state, data: [filtered], key });
+        subscriber.next({
+          state,
+          data: [
+            // workaround for serializing issues when sending DataFrame from web worker to the main thread
+            // DataFrame is making use of ArrayVectors which are es6 classes and thus not cloneable out of the box
```

Comment:
hoping to get rid of this soon, too: https://github.com/grafana/grafana/pull/38615

Reply:
nice, haven't seen that PR!